plugins/social.server.ts

```diff
@@ -1,5 +1,7 @@
 import { sendRedirect } from 'h3'
 
+const BOT_RE = /bot\b|index|spider|facebookexternalhit|baidu|crawl|wget|slurp|mediapartners-google/i
```

Comment:
```suggestion
const BOT_RE = /bot\b|index|spider|facebookexternalhit|baiduspider|crawl|wget|slurp|mediapartners-google/i
```

Reply:
in that case it should be caught by previous `spider`

Reply:
Baidu spider's UA always includes `baiduspider`. There's a browser made by Baidu. To avoid detecting the Baidu browser as a spider.

Reply:
Docs: https://help.baidu.com/question?prod_id=99&class=0&id=3001
<img width="434" alt="image" src="https://user-images.githubusercontent.com/6481596/212545645-755add82-df77-4bd3-8f01-035ac1e47fea.png">